### PR TITLE
Create Epicor.md

### PR DESCRIPTION
**Pull request by Tariqk26**
Created Connector Partner Guide

### CODE REVIEW

File: pages/v2/05-application-connectors/authentication/Epicor.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+---
+
+{::options parse_block_html="true" /}
+
+<section class="card">
+
+## Cyclr setup
+
+To authenticate your connector, you need the following information Epicor account.
```
Comment (joshuamullings) [inline pages/v2/05-application-connectors/authentication/Epicor.md:15]:
```suggestion
You need the following information to authenticate the Epicor connector:
```

File: pages/v2/05-application-connectors/authentication/Epicor.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+| Field            | Description                                                  |
+| ---------------- | ------------------------------------------------------------ |
+| Kinetic Server   | The Kinetic Server domain of your Epicor Kinetic account. this will be just domain without any HTTP protocals for example: `ukkineticserver00.epicorsaas.com` |
```
Comment (joshuamullings) [inline pages/v2/05-application-connectors/authentication/Epicor.md:19]:
```suggestion
| Kinetic Server   | The Epicor Kinetic Server account domain and subdomain not including the protocol. For example, if the Epicor Kinetic Server URL is `https://ukkineticserver00.epicorsaas.com` then enter `ukkineticserver00.epicorsaas.com` |
```

File: pages/v2/05-application-connectors/authentication/Epicor.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+| Kinetic Instance | The unique ID for the Instance of the Epicor Kinetic Server you wish to access. for example: `Test001Pilot` |
```
Comment (joshuamullings) [inline pages/v2/05-application-connectors/authentication/Epicor.md:20]:
```suggestion
| Kinetic Instance | The Epicor Kinetic Server unique instance ID. For example, `Test001Pilot`. |
```

File: pages/v2/05-application-connectors/authentication/Epicor.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+| API Key          | The API Key for your Epicor account. To request an API Key you will need to contact [Epicor Support](https://epiccare.epicor.com/epiccare) |
```
Comment (joshuamullings) [inline pages/v2/05-application-connectors/authentication/Epicor.md:21]:
```suggestion
| API Key          | The Epicor Kinetic account API key used to connect to the Epicor Kinetic Instance. To request an API Key, please contact [Epicor Support](https://epiccare.epicor.com/epiccare). |
```

File: pages/v2/05-application-connectors/authentication/Epicor.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+| Company ID       | The ID associated to the Company for the Epicor Kinetic account. |
+
+
+
+
+
+To set up the Epicor connector in Cyclr, go to your Cyclr console:
```
Comment (joshuamullings) [inline pages/v2/05-application-connectors/authentication/Epicor.md:28]: This section should follow the account setup format I think? It looks like the Epicor connector doesn't have any partner level parameters.

**Review by joshuamullings — CHANGES_REQUESTED** [review]

File: pages/v2/05-application-connectors/authentication/Epicor.md (hunk 5 above)
Comment (Tariqk26) [reply pages/v2/05-application-connectors/authentication/Epicor.md:28]: Correct, There's no Partner Level Parameters for the Epicor Connector as Each user has their own Server and instance.

Comment (BenCrum) [reply pages/v2/05-application-connectors/authentication/Epicor.md:28]: In which case, this isn't the correct format. 😊  The 1..8 points here relate to Partner-level parameters, e.g. "Go to "Connectors > Application Connector Library" is at the Partner-level, not at the point you're installing the Connector.

These problems should be resolved when I finish making a "Connector Guide where Partner-level parameters must be set in the Console" template and a "it's all entered when installing a Connector" template for Connector Guides to follow.

Closing this PR as moving into Confluence anyhoo.

**Comment by BenCrum** [comment]
Manually added to Confluence.

**Pull request closed without merging**